### ISSUE TITLE
Tolerance erroring out

### ISSUE THREAD

**Issue by capripot** [donv/capybara-screenshot-diff#74]
Adding the option `tolerance` without `vips`, I get this error:

```
 NoMethodError:
       undefined method `size' for #<Capybara::Screenshot::Diff::Drivers::ChunkyPNGDriver
```

Should we raise an error saying that `translate` option is incompatible with Chunky PNG driver (like noted in the README)?

**Comment by pftg** [comment]
@capripot yes will check it soon

**Comment by pftg** [comment]
@capripot as non-documented behavior added support of tolerance for ChunkyPNG, I'm going to update README later. Added some fixes in #77, could you check for your project it.
